clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRecord.java

```diff
@@ -258,7 +258,8 @@ public long offset() {
     }
 
     /**
-     * The timestamp of this record
+     * The timestamp of this record, in milliseconds
+     * (values are generally unix time, similar to System.currentTimeMillis())
```

Comment:
suggestion:

The timestamp of this record, in milliseconds elapsed since unix epoch.

Reply:
thanks, done!